app/src/main/java/com/github/palFinderTeam/palfinder/utils/image/ImageInstance.kt

```diff
@@ -58,8 +58,8 @@ data class ImageInstance(
                     imgStatus = FILE_NOT_FOUND
                     isCached = false
                 } finally {
+                    view.visibility = View.VISIBLE
```

Comment:
Can you explain why it works ? I'm not sure to understand

Reply:
The idea is that the view are recycled (because of the recycler view) and when a meetup or a user with no picture is displayed, the visibility is set to INVISIBLE, but when a picture is set, the visibility was not turned back to VISIBLE.

Reply:
ok thanks